include/fmt/chrono.h

```diff
@@ -451,6 +451,27 @@ FMT_END_DETAIL_NAMESPACE
 template <typename Char, typename Duration>
 struct formatter<std::chrono::time_point<std::chrono::system_clock, Duration>,
                  Char> : formatter<std::tm, Char> {
+  FMT_CONSTEXPR FMT_INLINE void generate_default_spec(char) {
+    this->specs = {"%Y-%m-%d %H:%M:%S", 17};
+  }
+  FMT_CONSTEXPR FMT_INLINE void generate_default_spec(wchar_t) {
+    this->specs = {L"%Y-%m-%d %H:%M:%S", 17};
+  }
+
+  template <typename ParseContext>
+  FMT_CONSTEXPR auto parse(ParseContext& ctx) -> decltype(ctx.begin()) {
+    auto it = ctx.begin();
+    if (it != ctx.end() && *it == ':') ++it;
+    auto end = it;
+    while (end != ctx.end() && *end != '}') ++end;
+    if (end == it) {
+      generate_default_spec(typename ParseContext::char_type{});
+    } else {
+      this->specs = {it, detail::to_unsigned(end - it)};
+    }
```

Comment:
nit: Please remove curly brackets around single statements for consistency with the rest of the code and apply clang-format.